### ISSUE TITLE
Hosted version / desktop app

### ISSUE THREAD

**Issue by Uzay-G** [archivy/archivy#289]
I am considering spending some time on a hosted version, because I'd like Archivy to be available to more non-technical people, and hosting it as a service would also make an integration with [Espial](https://github.com/Uzay-G/espial) easier.

I am also thinking of packaging Archivy as a single-file executable desktop app.

Opinions on this are welcome. Please tell me if you'd be interested in a hosted version.

**Comment by Surfer298** [comment]
This would be great! Could you also include a version for MacOSX? My wife uses a Macbook and is great fan of your project ;-)

**Comment by Uzay-G** [comment]
good to know, i'll try to get to this

**Comment by n00rm** [comment]
Hi, a hosted version is a good idea maybe to also fund the development of the project. But one thing that needs to be adressed first is the UI. Most people I talked to and showed them the App said: Concept great but UI/UX is terrible. I understand that the main priotity of early development is features and bug fixing and not UI/UX but to ship it as a "product" this should be adressed. Happy to talk about improvments.

**Comment by Uzay-G** [comment]
Yeah development has been somewhat stalling. Would be down to talk about this and am curious about improvements you'd be interested in.
